/usr/local/include/kcregex.h:35: error: comma at end of enumerator list

--- kcregex.h.orig	Wed Aug 29 14:56:59 2012
+++ kcregex.h	Wed Aug 29 14:57:13 2012
@@ -32,7 +32,7 @@ class Regex {
    */
   enum Option {
     IGNCASE = 1 << 0,                    ///< case-insensitive
-    MATCHONLY = 1 << 1,                  ///< matching only
+    MATCHONLY = 1 << 1                   ///< matching only
   };
   /**
    * Default constructor.
